frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -75,14 +85,31 @@ export class PieceEditor implements OnInit {
     this.pieceAPI.getAllPieces();
   }
 
+  private buildRenderingList() {
+    this.renderingItems = this.workingPieces.map((wpiece, wpieceIx) => {
```

Comment:
`(piece, pieceIndex) => {`

Reply:
Done. I prefer workingPiece to piece. Piece is a standard and ubiquitous data structure and WorkingPiece is a bigger structure than contains transient fields supporting the editing process.